open-apparel-registry/open-apparel-registry
# Enable custom styles

## Overview

Users should be able to select a color and a font in their embedded map configuration and have those settings applied throughout the application, as shown [here](https://app.abstract.com/projects/441633f0-7d8c-11e9-af1e-972cbddca60a/branches/272eff0b-3229-482f-a003-fad1e9558a87/commits/828df3ea1f6abe0ce18459a2dc1e5628e08e9df1/files/fc1791e3-b8d2-4182-b240-fef32fabe978/layers/88873DB4-D9AA-4DE4-9652-AC7ADB8C4147?collectionId=8ac19367-6932-43c2-95fa-7483af019a26&collectionLayerId=cfdc1a4d-1f4f-4fba-bcb0-4e4cb07bdaeb) and [here](https://app.abstract.com/projects/441633f0-7d8c-11e9-af1e-972cbddca60a/branches/272eff0b-3229-482f-a003-fad1e9558a87/commits/828df3ea1f6abe0ce18459a2dc1e5628e08e9df1/files/fc1791e3-b8d2-4182-b240-fef32fabe978/layers/02D8C4BE-87F3-4733-98A5-DD884FCE17D7?collectionId=8ac19367-6932-43c2-95fa-7483af019a26&collectionLayerId=1b494422-0155-4526-b816-fb927d1f5cbf).

Connects #1337, #1344

## Demo
<img width="1370" alt="Screen Shot 2021-05-21 at 2 07 35 PM" src="https://user-images.githubusercontent.com/21046714/119391235-1b11bf00-bc9c-11eb-9bfc-2861a374b7b0.png">
<img width="1368" alt="Screen Shot 2021-05-21 at 2 06 53 PM" src="https://user-images.githubusercontent.com/21046714/119390682-67103400-bc9b-11eb-8faa-74b8a80c116c.png">
<img width="1363" alt="Screen Shot 2021-05-21 at 3 43 17 PM" src="https://user-images.githubusercontent.com/21046714/119390703-6e374200-bc9b-11eb-9d3c-c0a12f15d2a5.png">
<img width="1437" alt="Screen Shot 2021-05-21 at 2 00 31 PM" src="https://user-images.githubusercontent.com/21046714/119390742-78f1d700-bc9b-11eb-9c01-c19b23a09530.png">

## Testing Instructions

* Navigate to http://localhost:6543/facilities and click through to a facility’s details. 
     - [x] The website should appear as usual
* Navigate to http://localhost:6543/?contributors=4&embed=1 (use a contributor with no embed config) and click through to a facility’s details
     - [x] The embedded map version of the site should appear as usual
* Login as c2@example.com
* Navigate to embed settings and select a custom font and color
* Navigate to http://localhost:6543/?contributors=2&embed=1 and click through to a facility’s details
     - [x] The website should have the selected color and font

## Checklist
